### PR TITLE
Regen bindings

### PR DESCRIPTION
**Pull request by Manishearth**
From https://hg.mozilla.org/mozilla-central/rev/d72cf6ecebaf

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/13672)

### CODE REVIEW

**Comment by Manishearth** [comment]
r? @emilio or @heycam 

don't r+, just mention r=me

File: components/style/binding_tools/regen.py [hunk 1]
```diff
@@ -230,7 +230,7 @@
             "StyleBasicShapeType", "StyleBasicShape", "nsCSSShadowArray",
             "nsINode", "nsIDocument", "nsIPrincipal", "nsIURI",
             "RawGeckoNode", "RawGeckoElement", "RawGeckoDocument",
-            "ServoNodeData",
+            "ServoNodeData", "nsString"
```
Comment (Manishearth) [inline components/style/binding_tools/regen.py]: it was pulling in some nsstring stuff, causing compile errors

**Comment by emilio** [comment]
r=me when needed

**Comment by Manishearth** [comment]
(incubator updated)

**Pull request merged**